xls/examples/hack.x

```diff
@@ -0,0 +1,258 @@
+// Copyright 2021 The XLS Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// An incomplete implementation of a the Hack CPU from the https://www.nand2tetris.org/ course.
+//
+// Implemented:
```

Comment:
I understand that there is a link for reference. Might be worth describing the instruction format, etc... for the reader.

Reply:
Done